doc/electrs-rest-btcd-1.0.0.yaml

```diff
@@ -0,0 +1,936 @@
+  version: 1.0.0
+  description: |
+    A Rust Implementation of the Electrum Server.
+    These docs are for the Standard Bitcoin Network Only.
+servers: 
+  - description: "Blockstream Public (bitcoin)" 
+    url: "https://blockstream.info/api/"
+  - description: "Blockstream Public (liquid)" 
+    url: "https://blockstream.info/liquid/api/"
+paths:
+  /blocks/tip/hash:
+    get:
+      tags:
+        - block
+      summary: Get the hash of the tip block
+      responses:
+        '200':
+          description: The hash of the tip block
+          content:
+            text/plain:
+              schema:
+                type: string
+  /blocks/tip/height:
+    get:
+      tags:
+        - block
+      summary: Get the height of the tip block
+      responses:
+        '200':
+          description: The height of the tip block
+          content:
+            text/plain:
+              schema:
+                type: string
+  /blocks/{start_height}:
+    get:
+      tags:
+        - block
+      summary: Get blocks starting from a specific height
+      parameters:
+        - name: start_height
+          in: path
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: List of blocks
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Block'
+  /block-height/{height}:
+    get:
+      tags:
+        - block
+      summary: Get block header by height
+      parameters:
+        - name: height
+          in: path
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: Block header
+          content:
+            application/json:
+              schema:
+                type: string
+  /block/{hash}:
+    get:
+      tags:
+        - block
+      summary: Returns information about a block
+      description: |
+          Elements-based chains have an additional `proof` field. <!--TODO: add link to block-format -->
+      parameters:
+        - name: hash
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Block data
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Block'
+  /block/{hash}/status:
+    get:
+      tags:
+        - block
+      summary: Get block status by hash
+      parameters:
+        - name: hash
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Block status
+          content:
+            application/json:
+              schema:
+                type: object
+  /block/{hash}/txids:
+    get:
+      tags:
+        - block
+      summary: Returns a list of Transaction ID's in a block.
```

Comment:
- transaction should be lower case, also no apostrophe on ID

Reply:
changed to lower case and removed apostrophie